plugin/src/main/kotlin/org/jlleitschuh/gradle/ktlint/PluginUtil.kt

```diff
@@ -143,10 +151,11 @@ internal fun String.androidVariantMetaFormatTaskName(
 /**
  * Get specific android variants from [BaseExtension].
  */
-internal val BaseExtension.variants: DomainObjectSet<out BaseVariant>? get() = when (this) {
-    is AppExtension -> applicationVariants
-    is LibraryExtension -> libraryVariants
-    is FeatureExtension -> featureVariants
-    is TestExtension -> applicationVariants
-    else -> null // Instant app extension doesn't provide variants access
-}
+internal val BaseExtension.variants: DomainObjectSet<out BaseVariant>?
```

Comment:
I have the official Kotlin code style applied and it forces me to have the `get()` in a new line.